subsys/usb/class/usb_dfu.c

```diff
@@ -631,6 +631,16 @@ static void dfu_status_cb(enum usb_dc_status_code status, const u8_t *param)
 	}
 }
 
+#ifdef CONFIG_USB_COMPOSITE_DEVICE
+static void dfu_status_composite_cb(struct usb_cfg_data *cfg,
+				    enum usb_dc_status_code status,
+				    const u8_t *param)
+{
+	ARG_UNUSED(cfg);
+	dfu_status_cb(status, param);
+}
+#endif
+
```

Comment:
This is fixed already in PR  #13310

https://github.com/zephyrproject-rtos/zephyr/pull/13310/commits/539b2754bf0f182dffd5678b4e58b5e326e7e657#diff-98a1f9ecb2c01757bdb4413acdfcab02L595